custom_components/waste_collection_schedule/waste_collection_schedule/source/goessendorf_at.py

```diff
@@ -0,0 +1,52 @@
+from datetime import datetime
+
+import requests
+from bs4 import BeautifulSoup
+from waste_collection_schedule import Collection  # type: ignore[attr-defined]
+
+TITLE = "Marktgemeinde Gössendorf"
+DESCRIPTION = "Source for Marktgeneinde Gössendorf, AT"
+URL = "https://www.goessendorf.com/"
+TEST_CASES = {"TestSource": {}, "IgnoredArgument": {"_": ""}}
+ICON_MAP = {
+    "Bioabfall": "mdi:food",
+    "Altpapier P1": "mdi:newspaper",
+    "Altpapier P2": "mdi:newspaper",
+    "Sperrmüll S1": "mdi:factory",
+    "Sperrmüll S2": "mdi:factory",
+    "Restmüll R1": "mdi:trash-can",
+    "Restmüll R2": "mdi:trash-can",
+}
+
+
+class Source:
+    def __init__(self, _=None):
```

Comment:
You don't need the dummy argument, it just makes configuration more confusing (The HA config GUI shows this field)